src/o365/spo/commands/list/list-view-list.spec.ts

```diff
@@ -0,0 +1,2260 @@
+                  "Id":"e021e923-0801-4a16-9775-545239356739",
+                  "ImageUrl":null,
+                  "IncludeRootFolder":false,
+                  "ViewJoins":null,
+                  "JSLink":"clienttemplates.js",
+                  "ListViewXml":"<View Name=\"{E021E923-0801-4A16-9775-545239356739}\" MobileView=\"TRUE\" Type=\"HTML\" Hidden=\"TRUE\" DisplayName=\"\" Url=\"/sites/ninja/SitePages/Home.aspx\" Level=\"1\" BaseViewID=\"50\" ContentTypeID=\"0x\" ><Query><OrderBy><FieldRef Name=\"Modified\" Ascending=\"FALSE\" /></OrderBy></Query><ViewFields><FieldRef Name=\"DocIcon\" /><FieldRef Name=\"LinkFilename\" /></ViewFields><RowLimit Paged=\"TRUE\">15</RowLimit><JSLink>clienttemplates.js</JSLink><XslLink Default=\"TRUE\">main.xsl</XslLink><Toolbar Type=\"Standard\"/></View>",
+                  "Method":null,
+                  "MobileDefaultView":false,
+                  "MobileView":true,
+                  "ModerationType":null,
+                  "NewDocumentTemplates":null,
+                  "OrderedView":false,
+                  "Paged":true,
+                  "PersonalView":false,
+                  "ViewProjectedFields":null,
+                  "ViewQuery":"<OrderBy><FieldRef Name=\"Modified\" Ascending=\"FALSE\" /></OrderBy>",
+                  "ReadOnlyView":false,
+                  "RequiresClientIntegration":false,
+                  "RowLimit":15,
+                  "Scope":0,
+                  "ServerRelativePath":{
+                    "__metadata":{
+                      "type":"SP.ResourcePath"
+                    },
+                    "DecodedUrl":"/sites/ninja/SitePages/Home.aspx"
+                  },
+                  "ServerRelativeUrl":"/sites/ninja/SitePages/Home.aspx",
+                  "StyleId":null,
+                  "TabularView":true,
+                  "Threaded":false,
+                  "Title":"",
+                  "Toolbar":"",
+                  "ToolbarTemplateName":null,
+                  "ViewType":"HTML",
+                  "ViewData":null,
+                  "VisualizationInfo":null
+              }
+            ]
+          });
+        }
+      }
+
+      return Promise.reject('Invalid request');
+    });
+
+    auth.site = new Site();
+    auth.site.connected = true;
+    auth.site.url = 'https://contoso-admin.sharepoint.com';
+    auth.site.tenantId = 'abc';
+    cmdInstance.action = command.action();
+    cmdInstance.action({
+      options: {
+        debug: true,
+        listTitle: 'Documents',
+        webUrl: 'https://contoso.sharepoint.com/sites/ninja'
+      }
+    }, () => {
+      try {
+        assert(cmdInstanceLogSpy.calledWith([
+          {
```

Comment:
The data bellow does not seem related with the data you have above since I cannot find `"StringId": "0x010100260C61709CD8E548948F9BF605F8F54F",` in the faked response that starts at line `115`.

The way I do it is. I know that the last call will have the result so I use the `cmdInstanceLogSpy.lastCall` to speed up the execution of the test.
You can place `console.log(JSON.stringify(cmdInstanceLogSpy.lastCall.args));` or `cmdInstanceLogSpy.getCalls().forEach(x => {console.log(JSON.stringify(x.args));});` to see all the calls output when you run `npm test`. This will give you clear idea what you can assert.

I would replace your 
```
assert(cmdInstanceLogSpy.calledWith([
          {
            "StringId": "0x010100260C61709CD8E548948F9BF605F8F54F",
            "Name": "Document",
            "Hidden": false,
            "ReadOnly": false,
            "Sealed": false
          },
          {
            "StringId": "0x0120000EAD53EDAD7C6647B0D976EEC953F99E",
            "Name": "Folder",
            "Hidden": false,
            "ReadOnly": false,
            "Sealed": true
          }
        ]));
```

with something like:

```
const result: any = cmdInstanceLogSpy.lastCall.args[0].value;
        
assert.equal(result[0].Id, '85907fb0-cda1-43ec-832e-27118c89cf9f');
assert.equal(result[1].Id, '281e80fd-099b-4eba-b622-a94fa03bf865');
```

or 

```
const result: any = cmdInstanceLogSpy.lastCall.args[0].value;
        
assert.equal(result[0].Id, '85907fb0-cda1-43ec-832e-27118c89cf9f');
assert.equal(result[1].Id, '281e80fd-099b-4eba-b622-a94fa03bf865');
```

or just

```
assert.equal(cmdInstanceLogSpy.lastCall.args[0].value.length, 5);
```